src/components/CourseForm.jsx

```diff
@@ -3,7 +3,7 @@ import { Button, Modal, Form} from 'react-bootstrap';
 
 
 async function addCourse(course){
-    return fetch('url',{
+    return fetch('http://localhost:8080/api/courses/addCourse',{
```

Comment:
okay for now.. but in the future you'll want to use an environment variable for this url. Then you can deploy the same code base and point it a the prod API with an environment variable

e.g.
```js
const baseUrl = process.env.API_BASE_URL || 'http://localhost:8080`;
return fetch(`${baseUrl}/courses/addCourse`);
```